### ISSUE TITLE
Pretraining method

### ISSUE THREAD

**Issue by oggyfaker** [usefulsensors/moonshine#42]
Hi i am really curious about the loss function you use for train pretrain-weight (CTC or Cross-Entropy loss ?) . It not be mentioned in paper. 
Hope to see your answer !
